### PR TITLE
JFG-986: remove RpsClockConfiguration

### PR DESCRIPTION
**Pull request by anton-antonenko**

### CODE REVIEW

File: chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/load/JLoadProfileRps.java [hunk 1]
```diff
@@ -16,33 +16,71 @@
...
-     * @param maxLoadThreads - The maximum number of threads, which Jagger engine can create to provide the requested load
+     * @details Constructor parameters are mandatory for the JLoadProfileRps. All parameters, set by setters are optional
+     * @n
+     * @param requestsPerSecond   - The number of requests per second Jagger shall perform
+     * @param maxLoadThreads      - The maximum number of threads, which Jagger engine can create to provide the requested load
      * @param warmUpTimeInSeconds - The warm up time value in seconds. Jagger increases load from 0 to @b requestPerSecond by @b warmUpTimeInSeconds
      */
-    public static JLoadProfileRps of(RequestsPerSecond requestsPerSecond, MaxLoadThreads maxLoadThreads, WarmUpTimeInSeconds warmUpTimeInSeconds) {
-        return new JLoadProfileRps(requestsPerSecond, maxLoadThreads, warmUpTimeInSeconds);
+    public static Builder builder(RequestsPerSecond requestsPerSecond, MaxLoadThreads maxLoadThreads, WarmUpTimeInSeconds warmUpTimeInSeconds) {
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/load/JLoadProfileRps.java]: maxloadthreads & warmuptime are also oprional

see: com.griddynamics.jagger.xml.beanParsers.task.TpsDefinitionParser
for previous usage and default values

File: chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/load/JLoadProfileRps.java [hunk 2]
```diff
@@ -16,33 +16,71 @@
...
-    public JLoadProfileRps(RequestsPerSecond requestsPerSecond, MaxLoadThreads maxLoadThreads, WarmUpTimeInSeconds warmUpTimeInSeconds) {
-        Objects.nonNull(requestsPerSecond);
-        Objects.nonNull(maxLoadThreads);
-        Objects.nonNull(warmUpTimeInSeconds);
-        
-        this.requestsPerSecond = requestsPerSecond.value();
-        this.maxLoadThreads = maxLoadThreads.value();
-        this.warmUpTimeInSeconds = warmUpTimeInSeconds.value();
+    private JLoadProfileRps(Builder builder) {
+        Objects.nonNull(builder);
```
Comment (a-badaev) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/load/JLoadProfileRps.java]: Objects.requireNonNull

File: chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/load/JLoadProfileRps.java [hunk 3]
```diff
@@ -16,33 +16,71 @@
...
+         * @n
+         * @details Constructor parameters are mandatory for the JLoadProfileRps. All parameters, set by setters are optional
+         * @n
+         * @param requestsPerSecond   - The number of requests per second Jagger shall perform
+         * @param maxLoadThreads      - The maximum number of threads, which Jagger engine can create to provide the requested load
+         * @param warmUpTimeInSeconds - The warm up time value in seconds. Jagger increases load from 0 to @b requestPerSecond by @b warmUpTimeInSeconds
+         */
+        public Builder(RequestsPerSecond requestsPerSecond, MaxLoadThreads maxLoadThreads, WarmUpTimeInSeconds warmUpTimeInSeconds) {
+            Objects.nonNull(requestsPerSecond);
+            Objects.nonNull(maxLoadThreads);
```
Comment (a-badaev) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/load/JLoadProfileRps.java]: Objects.requireNonNull

File: chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/load/JLoadProfileRps.java [hunk 4]
```diff
@@ -16,33 +14,93 @@
...
+         * @return JLoadProfileRps object.
+         */
+        public JLoadProfileRps build() {
+            return new JLoadProfileRps(this);
+        }
+
+        /** Optional: Tick interval (in ms). Default is 1000 ms.
+         * @param tickInterval Tick interval of load
+         */
+        public Builder withTickInterval(int tickInterval) {
```
Comment (dlatnikov) [inline chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/load/JLoadProfileRps.java]: it should be withTickIntervalInSeconds
- not milliseconds

**Pull request merged**